cmd/zetae2e/config/local.yml

```diff
@@ -16,10 +16,14 @@ additional_accounts:
     bech32_address: "zeta13t3zjxvwec7g38q8mdjga37rpes9zkfvv7tkn2"
     evm_address: "0x8Ae229198eCE3c889C07DB648Ec7C30E6051592c"
     private_key: "105460aebf71b10bfdb710ef5aa6d2932ee6ff6fc317ac9c24e0979903b10a5d"
-  user_bitcoin:
+  user_bitcoin1:
```

Comment:
Would be better to be as precise as possible: bitcoin_deposit, bitcoin_withdraw

Reply:
https://github.com/zeta-chain/node/pull/3105/commits/03b7c9bc1c723a4924feb4490382a2a02cdb16d9